package.json

```diff
@@ -31,7 +31,7 @@
   "homepage": "https://github.com/ExtensionEngine/boutique#readme",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.1.2",
+    "@babel/core": "^7.2.2",
```

Comment:
Hm, you shouldn't update stuff you don't really need to because it might introduce bugs elsewhere ⚠️